tests/test_exporter.py

```diff
@@ -0,0 +1,112 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import unittest
+
+import pyarrow
+from lsst.dax.obscore import DatasetTypeConfig, ExporterConfig, ObscoreExporter
+from lsst.dax.obscore.obscore_exporter import _STATIC_SCHEMA
+
+
+class TestCase(unittest.TestCase):
```

Comment:
I assume it would be helpful to have a small butler repo that we can run tests on? It just needs the registry part so should be fairly easy to extract a few rows from an existing repo and import them for a test.

Reply:
That would be useful, indeed, I'll try to add it.

Reply:
Added a unit test which uses some data exported from ci_hsc_gen3 repo. Importing that into a test repo and exporting to a parquet file.